app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -985,6 +945,7 @@ private val Event.wrapper: EventWrapper<*>?
         is Event.AddonInstalled -> null
         is Event.SearchWidgetInstalled -> null
         is Event.SyncAuthFromSharedReuse, Event.SyncAuthFromSharedCopy -> null
+        else -> null
```

Comment:
we shouldn't need this if the events are removed.  Is there an event that we did not remove?

Reply:
The events that weren't removed are `RemoveBookmark`, `RemoveBookmarkFolder` and `RemoveBookmarks`. They are used to handle the [type of bookmark removed](https://github.com/mozilla-mobile/fenix/blob/ada6a74d6709a9c6bb5898c0fd033af3a92eb866/app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragmentInteractor.kt#L92). Would it be better to move them to `BookmarkController`?

Reply:
I see, do we need to create a new enum to replace these 3 events?  Since I assume we will be removing `Event.kt` in the future after we removed the need for the wrapper?